articles/faq/profile-info.md

```diff
@@ -1,12 +1,8 @@
 ---
-title: "Where do I find my Username, Account ID (Customer ID), or Email Address?"
-description: "When you're submitting a Support ticket or communicating with us via our community Slack channel we ask you to identify your account by providing either your Username Account ID Customer ID or Email Address used to create the Spark Post account Having this info allows us to help you faster..."
+title: "Where do I find my username, account ID, or account email address?"
+description: "Sometimes when we communicate with you as a user, we ask you to identify your account by providing your username, account ID, or account email address. Having this info allows us to help you faster in situations where we don't already have these details about who you are."
 ---
 
-When you're submitting a Support ticket or communicating with us via our [community Slack channel](http://slack.sparkpost.com), we ask you to identify your account by providing either your Username, Account ID (Customer ID), or Email Address used to create the SparkPost account. Having this info allows us to help you faster, especially if you are sending us an email from an address not associated with your account.
+Sometimes when we communicate with you as a user (e.g. via our [community Slack channel](http://slack.sparkpost.com)), we ask you to identify your account by providing your username, account ID (Customer ID), or account email address. Having this info allows us to help you faster in situations where we don't already have these details about who you are.
```

Comment:
Should we remove all instances of "(Customer ID)" since we removed it from the title and description?

Reply:
I'm ok with leaving it in the body. :)